spec/controllers/spree/admin/back_in_stock_notifications_controller_spec.rb

```diff
@@ -38,6 +38,16 @@
           end
         end
       end
+
+      context "format CSV" do
+        subject { get :index, params: { format: :csv } }
+
+        let!(:back_in_stock_notification) { create(:back_in_stock_notification) }
+
+        it "does not raise an error" do
+          expect{ subject }.to_not raise_error
```

Comment:
Might be worth writing a more robust test than this. 😄

Reply:
Ok! I have added a test for the contents. I didn't realise it was easy to get via `response.body`